tsuru/client/env.go

```diff
@@ -35,16 +35,25 @@ Example:
 
   tsuru env-set NAME=value OTHER_NAME="value with spaces" ANOTHER_NAME='using single quotes' -p`
 
+const MissingAppOrJob = "You must pass an application or job"
```

Comment:
just a cosmetic request, please prefix these with Err...
 ```suggestion
const ErrMissingAppOrJob = "You must pass an application or job"
```